config/locales/en.yml

```diff
@@ -32,7 +32,7 @@ en:
           one: "Showing <b>1</b> %{entry_name}"
           other: "Showing <b>%{count}</b> %{entry_name}"
       more_pages:
-        display_entries: "Showing <b>%{first}&ndash;%{last}</b> of <b>%{total}</b> %{entry_name}"
+        display_entries: "<b>Preview of records %{first} &ndash; %{last}</b>"
```

Comment:
I am not sure that this text is strictly accurate since.:
* Records has no ordering
* We provide a default display order of either `key` or `name`.

Perhaps we should just say *showing* as before, or maybe just 
```<b>Preview of records %{last} records</b>```
?

CC @mpwoods

Reply:
Let's go with 'Preview of 5 records'

Reply:
Changed; I've also changed it to this phrasing for the register that are able to load more records . Where there are less than the limit it'll say 'Showing 3 records', since that isn't really a preview.